### DISCUSSION TITLE
How does autobump work, and why isn't openjdk automatically updated ?

### DISCUSSION THREAD

**Discussion by claudionieder** [Homebrew/discussions#5304] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.18-74-g401500e
ORIGIN: https://github.com/Homebrew/brew
HEAD: 401500e70a2eba52fa59f3c3396d33e7c4dd3e96
Last commit: 16 hours ago
Core tap HEAD: 39a3c7d86efad8731ad27cb2fa146b026b6d6bfb
Core tap last commit: 2 hours ago
Core tap JSON: 20 Apr 12:57 UTC
Core cask tap HEAD: 8603e43f2f793a178bcffa8be2489522a0a47a6b
Core cask tap last commit: 22 minutes ago
Core cask tap JSON: 20 Apr 12:57 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: vim
HOMEBREW_GITHUB_API_TOKEN: set
HOMEBREW_MAKE_JOBS: 10
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 3.1.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.44.0 => /opt/homebrew/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 14.4.1-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: 15.3
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

I wonder about this autobump feature. When I do a brew bump openjdk I get the answer
```
==> openjdk
Formula is on autobump list.
```
Yet, openjdk is still at version 21.0.2 though the most recent ones are either 21.0.3 or 22.0.1, depending on whether openjdk is supposed to represent the latest available release or the latest LTS release.

What needs to be done so that openjdk can be updated ?

**Comment by SMillerDev** [comment]
> What needs to be done so that openjdk can be updated ?

For the build and tests to complete successfully in the pull request

**Reply by maxwelbm** [reply]
Hi @SMillerDev,

I would like to know if there is any problem in leaving my CLI in Autobump without worrying. It was recently added and is working great, but I have some doubts about the process.

Are there any specific criteria for a CLI to be included in Autobump? And are there cases where it would be better to remove a formula from it? I want to understand if this configuration is the most suitable for my project or if there is something else I should consider.

Thank you in advance for your attention!

**Reply by p-linnane** [reply]
Autobump just means our workflow will open a PR within a few hours of a new version being released. Most of the time new versions build just fine and are merged. If the build fails, a maintainer will review and attempt to solve the issue. Barring that, we will reach out to upstream to request guidance.
